### ISSUE TITLE
Best practices for handing tab delimited files using fastcsv?

### ISSUE THREAD

**Issue by AnnanFay** [draftcode/fastcsv#8]
I have many large _tab_ seperated files which I need to parse with python. I found fastcsv and it seems like what I need, except for using commas.

Is there an efficient way to wrap an io.open stream to replace tabs with commas? I'm not familiar with the API.

**Comment by draftcode** [comment]
This library was originally written for reading non-ASCII csv in Python2. In Python2, there were two string-ish types. At that time, the standard csv module was based on str and it's not non-ASCII compatible. To read those non-ASCII csv files, I had to do three decoding/encoding processes. I explain that here: http://pythonhosted.org/fastcsv/#why-not-use-the-standard-csv-module

Now, pretty much everyone uses Python3, and Python3's csv module doesn't have that codec issue. Just try using the standard csv module. It's been 5 years since I measured the performance (in Python2) and the current library might do better than this. Also nowadays Python is used for data processing, so I imagine there's better data parsing library.
